pymc/variational/opvi.py

```diff
@@ -957,9 +957,9 @@ def __init_group__(self, group):
         if not group:
             raise GroupError("Got empty group")
         if self.local:
```

Comment:
I think the context of the issue was to entirely delete the code and `local` kwarg, feel free to just delete large portions of the code.